common/main/en.xml

```diff
@@ -331,6 +348,7 @@ annotations.
 			<language type="kut">Kutenai</language>
 			<language type="kv">Komi</language>
 			<language type="kw">Cornish</language>
+			<language type="kwk">Kwakʼwala</language>
```

Comment:
Several languages have different names in ISO, what is the best resource to check against?

Reply:
Wikipedia and SIL are reasonable sources. (SIL has a paywall though, so you can't hit it many times a month). 
Note: we can always change names later.